spec/dummy/spec/helpers/react_on_rails_helper_spec.rb

```diff
@@ -74,6 +74,21 @@
       it { is_expected.not_to include react_component_div }
       it { is_expected.to include react_definition_div }
     end
+
+    context "with skip_display_none option" do
+      before { ReactOnRails.configuration.skip_display_none = true }
+
+      let(:react_definition_div_skip_display_none) do
+        "<div class=\"js-react-on-rails-component\"
+              data-component-name=\"App\"
+              data-props=\"{&quot;name&quot;:&quot;My Test Name&quot;}\"
+              data-trace=\"false\"
+              data-expect-turbolinks=\"true\"
+              data-dom-id=\"#{id}\"></div>".squish
+      end
+
+      it { is_expected.to include react_definition_div_skip_display_none }
```

Comment:
I believe the test above it should cover the normal case as it defaults to display: none, I didn't want to repeat the same line but I can add a test for default.

https://github.com/shakacode/react_on_rails/blob/TangRufus-add-skip-display-none/spec/dummy/spec/helpers/react_on_rails_helper_spec.rb#L50